### PR TITLE
Append .co-external-link icon to link using psuedo element and make it an inline-block

### PR DESCRIPTION
**Pull request by sg00dwin**
... this enables it to stay appended to the text and won't wrap by itself. Also the `:hover` underline is limited to text.

Fixes https://jira.coreos.com/browse/CONSOLE-488

**before**
screen shot 2018-05-29 at 1 38 42 pm

**after**
screen shot 2018-05-29 at 1 39 09 pm

**before**
screen shot 2018-05-29 at 11 31 53 am

**after**
screen shot 2018-05-29 at 11 32 24 am

**before**
screen shot 2018-05-29 at 11 34 39 am

**after**
screen shot 2018-05-29 at 11 34 13 am

### CODE REVIEW

**Comment by ggreer** [comment]
Couldn't this be accomplished more simply by using a selector like `a[target="_blank"]`? That would also make it less likely that future external links would be styled incorrectly. (It's pretty easy to forget to add the `co-external-link` class.)

File: frontend/public/style/_common.scss [hunk 1]
```diff
@@ -190,3 +190,21 @@ input[type=number]::-webkit-outer-spin-button {
 .co-break-word {
   @include co-break-word;
 }
+
+// append external-link icon to <a> so that it doesn't wrap without text
+// there must be no white space between the text and the closing </a> tag holding the pseudo element
+.co-external-link:after {
+  content: "\f08e";
```
Comment (spadgett) [inline frontend/public/style/_common.scss]: Does `$fa-var-external-link` work?

Comment (ggreer) [reply frontend/public/style/_common.scss]: Yeah, lines like these are totally opaque. What icon is it referencing? Who knows?

**Comment by ggreer** [comment]
Good point about anchor tags in buttons. In that case, I think it'd be best to make an external link component. This usage of classes seems better than what's in master though, so I'm OK with this implementation.

One change in behavior is that since the entire anchor tag is inline-block, it will wrap as a block. There's no easy way to get word wrap while preventing just the icon from wrapping, so... :man_shrugging:

**Pull request merged**